drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -12,6 +12,20 @@ function DragAndDropBlock(runtime, element, configuration) {
     var state = undefined;
     var __vdom = virtualDom.h();  // blank virtual DOM
 
+    // Keyboard accessibility
+    var CTRL = 17;
+    var ESC = 27;
+    var RET = 13;
+    var SPC = 32;
+    var TAB = 9;
+    var M = 77;
```

Comment:
@itsjeyd I believe it is only use when "dragging" an item into a zone using keyboard, in "Ctrl+M" chord, as an alternative to just hitting space or return. And this feature is not mentioned in Keyboard Help. I woould suggest either adding "Ctrl+M" to keyboard help or remove this `M` altogether.